css/properties/grid-column-start.json

```diff
@@ -105,9 +105,14 @@
             "safari_ios": {
               "version_added": "10.3"
             },
-            "samsunginternet_android": {
-              "version_added": "6.0"
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "6.0"
+              },
+              {
+                "version_added": true
```

Comment:
Can we get a comment to why there are two version_added?

If it is because it is behind a flag I think it can be reverted.